### ISSUE TITLE
Is there a way to find all imports in a folder?

### ISSUE THREAD

**Issue by skjerns** [mgedmin/findimports#5]
I want to display all dependencies of scripts in one folder, including subfolders.

How can I display them, without having them listed per file?

**Comment by skjerns** [comment]
Yes, but my problem is that they are listed for each script separately.
What I would need is a list imports that are done overal, not per script

**Comment by mgedmin** [comment]
Then I'm not sure findimports can help you...

**Comment by jimklo** [comment]
it seems like with a little shell magic you could do this...

Wouldn't something like this work do what you're looking for?

```bash
findimports ./ | grep -v ":" | sed 's/\..*$//g' | sort | uniq
```

The above can be augmented to work with `find` as well.
